test/js/pageobjects/Welcome.js

```diff
@@ -1,29 +1,35 @@
 import Base from './BasePage.js'
-import {locale, verifyElementCopy} from '../utils/mochaw'
+import { locale, verifyElementCopy } from '../utils/mochaw'
+import { testFocusManagement, elementCanReceiveFocus } from '../utils/accessibility'
 
 class Welcome extends Base{
     getPrimaryBtn() { return this.$('.onfido-sdk-ui-Button-button'); }
     get welcomeTitle() { return this.$('.onfido-sdk-ui-PageTitle-titleSpan')}
     get welcomeSubtitle() { return this.$('.onfido-sdk-ui-Welcome-text')}
     get footer() { return this.$('.onfido-sdk-ui-Theme-footer')}
     get primaryBtn() { return this.$('.onfido-sdk-ui-Button-button')}
-    
+
     copy(lang) {return locale(lang) }
-    
+
     verifyTitle (copy) {
         const welcomeScreenStrings = copy.welcome
         verifyElementCopy(this.welcomeTitle, welcomeScreenStrings.title)
         return this.welcomeTitle
     }
 
+    async verifyFocusManagement() {
+      await testFocusManagement(this.welcomeTitle, this.driver)
+    }
+
     verifySubtitle(copy) {
```

Comment:
@bartzet did you forget to include `async` in these `verify...` functions or are these new or something?

Reply:
yeah, missed that screen 🙈 
Already added in the PR for the Cross-device sync tests.